extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -377,15 +376,11 @@ export class SubscriptionService {
 
       if (this.viewTrackerPromise_) {
         this.viewTrackerPromise_.then(() => {
+          return this.platformStore_.getGrantEntitlement();
+        }).then(grantStateEntitlement => {
           const localPlatform = this.platformStore_.getLocalPlatform();
-
-          if (selectedPlatform.isPingbackEnabled()) {
-            selectedPlatform.pingback(selectedEntitlement);
-          }
-
-          if (selectedPlatform.getServiceId() !== localPlatform.getServiceId()
-              && localPlatform.isPingbackEnabled()) {
-            localPlatform.pingback(selectedEntitlement);
+          if (localPlatform.isPingbackEnabled()) {
+            localPlatform.pingback(grantStateEntitlement);
```

Comment:
Let's protect (and test) the possibility that `grantStateEntitlement` could be `null`.

Reply:
`getGrantEntitlement` does not resolve without an `Entitlement` so this should be fine